Gemfile

```diff
@@ -78,6 +78,7 @@ group :test do
   gem "rake"
   gem "selenium-webdriver"
   gem "simplecov"
+  gem "docx"
```

Comment:
Is this gem being used because it has functionality that sablon does not, or is it just different/easier to use?

Reply:
yeah it's for testing rather than rendering--sablon is for the word document templates, this is for checking that the word document generated correctly with sablon